Source/WebKit/Platform/IPC/unix/ConnectionUnix.cpp

```diff
@@ -43,26 +43,17 @@
 #include <wtf/TZoneMallocInlines.h>
 #include <wtf/UniStdExtras.h>
 
-#if USE(GLIB)
-#include <gio/gio.h>
-#include <wtf/glib/GUniquePtr.h>
-#endif
-
 #if OS(DARWIN)
 #define MSG_NOSIGNAL 0
 #endif
```

Comment:
```suggestion
```

Probably safe to delete this because: I think only WebKitGTK built this file on Darwin, and now that will never happen anymore. And if I'm wrong, then the build should fail, somebody should complain, and we can just put it back.

Reply:
(Feel free to ignore this if you don't want to change it.)

Reply:
Yes, I thought that if someone is building WebKitGTK on darwing they could just pick ConnectionUnix at build time.